publisher.rb

```diff
@@ -1,25 +1,46 @@
 require 'aws-sdk-sqs'
 require_relative 'load_aws'
 require_relative 'aws_sqs_client'
+require_relative 'q_it_errors'
 
 class Publisher
   
   attr_reader :sqs, :sleep_period, :queue_name
 
   def initialize(options)
     @sqs = AwsSQSClient.new.client
-    @sleep_period = valid_sleep_period?(options[1]) ? options[1].to_i : 5
-    @queue_name = options[0]
+    @queue_name = valid_queue_name(options[0])
```

Comment:
You could name the validation methods starting with `validate` - sounds more apt to the action carried out.

Reply:
The reason behind naming it as `valid_xyz` is we return the value and not `true` / `false`. Had it been so, validate would make sense. Anyway, moving all this to app.rb to validate before obj creation, so will change a few things here.